### PR TITLE
[Fix] Support MMDet custom dataset

### PR DESCRIPTION
**Pull request by SingleZombie**
## Motivation

#22 : We cannot get the `CLASSES` of custom dataset in MMDet.

## Modification

Modify `object_detection_model.py::get_classes_from_cfg`.

### CODE REVIEW

**Comment by SingleZombie** [comment]
@hhaAndroid Hi, could you help us review the PR? Seems we cannot invite you as a reviewer.

File: tests/test_codebase/test_mmdet/test_object_detection_model.py [hunk 1]
```diff
@@ -385,34 +385,43 @@ def partition1_postprocess(self, *args, **kwargs):
...
+    data_cfg4 = mmcv.Config(dict(data=dict(error=dict(type='CocoDataset'))))
+
+    @pytest.mark.parametrize('cfg',
+                             [data_cfg1, data_cfg2, data_cfg3, data_cfg4])
+    def test_get_classes_from_cfg(self, cfg):
+        from mmdet.datasets import DATASETS
+        from mmdeploy.codebase.mmdet.deploy.object_detection_model import \
+            get_classes_from_config
+
+        if 'error' in cfg.data:
```
Comment (hhaAndroid) [inline tests/test_codebase/test_mmdet/test_object_detection_model.py:411]: I don't understand where this error comes from？

Comment (SingleZombie) [reply tests/test_codebase/test_mmdet/test_object_detection_model.py:411]: In `data_cfg4`. The key of inner dict is "error".

**Review by RunningLeon — APPROVED** [review]
LGTM

**Review by hhaAndroid — APPROVED** [review]

**Pull request merged**